src/TensorKitSectors.jl

```diff
@@ -18,11 +17,12 @@ export FusionStyle, UniqueFusion, MultipleFusion, SimpleFusion, GenericFusion,
 export BraidingStyle, NoBraiding, SymmetricBraiding, Bosonic, Fermionic, Anyonic
 export SectorSet, SectorValues, findindex
 export rightone, leftone
+export S_matrix, T_vector
```

Comment:
I'm not necessarily opposed to adding these functions, I just wanted to ask if they are useful in TensorKit itself?

The main reason this package is a separate one is to have a lightweight dependency in order to define new types, so the amount of functionality that is here is deliberately kept smaller. Obviously these are small functions so this should be fine, I was just wondering if there is a better place for this that might make more sense.

In any case, if we decide on adding this here, we have to add docstrings and make sure that it is somehow clear that this is also part of the interface.